Makefile

```diff
@@ -134,8 +134,22 @@ binary-containerized: vendor
 ###############################################################################
 
 ## Builds the code and runs all tests.
+# ensure we confirmation
```

Comment:
Is this a copy-paste error? Seems like we're re-defining the `imagetag` target?

Reply:
Um, maybe? Possibly? Yeah. :-)

Reply:
Fixing now.

Reply:
Fixed. Thanks for finding.